nationality-park/style.scss

```diff
@@ -0,0 +1,160 @@
+/*MAIN SETTINGS*/
+$backgroundColor: rgb(30, 30, 30);
+$backgroundColor50: rgba(30, 30, 30, 0.5);
+$accentColor: green;
+$textColor: whitesmoke;
+*{
+    margin: 0;
+    padding: 0;
+    color: $textColor;
+    font-family: Tahoma, sans-serif;
+}
+body{
+    background-color: $backgroundColor;
+}
+.container{
+    width: 100%;
+    height: auto;
+    position: relative;
+}
+.contentWrapper{
+    width: 80%;
+    height: 100%;
+    margin: auto;
+    z-index: 10;
+    position: relative;
+}
+.accentText{
+    background-color: $accentColor;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    display: inline-block;
+    padding: 10px 15px 10px 15px;
```

Comment:
optimise it with 

padding: 10px 15px;

https://www.w3schools.com/cssref/pr_padding.php